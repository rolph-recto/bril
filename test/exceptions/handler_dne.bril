# handler label does not exist
main {
  v: int = const 4;
  v2: int = const 5;
  v3: int = const 10;
  handle explosion explosionLabel2;
  call func1 v;
  call func1 v2;
  ret;
explosionLabel:
  print v3;
  ret;
}

func1 x:int {
  v: int = const 1;
  throw explosion;
  print x;
  ret;
}
