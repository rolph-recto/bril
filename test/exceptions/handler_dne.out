label explosionLabel2 not found
